services/chat.js

```diff
@@ -11,6 +13,22 @@ const chatService = {
         { user: chatMember, role: chatMemberRole }
       ]
     })
+  },
+  getChats: async (currentUser) => {
+    const { id: user, role: userRole } = currentUser
+
+    return await Chat
+      .find({
+        'members.user': mongoose.Types.ObjectId(user),
+        'members.role': userRole
+      })
+      .populate({
+        path: 'latestMessage',
+        populate: {
+          path: 'author',
+          select: '_id firstName lastName'
```

Comment:
This populate is wrong.
When a user sends a message, they will appear in the list of the chat. This means that if a user has sent a message in multiple chats, they will see themselves listed in each chat.

Reply:
We should see the opposite user.